docs/upgrade_with_cli.md

```diff
@@ -0,0 +1,51 @@
+# Upgrade Percona Everest using the CLI
+
+Percona Everest regularly releases updates that contain bug fixes, security improvements, and other enhancements to improve the overall performance of your database.
+
+
+## Before you upgrade
+
+!!! note
+     Ensure that you have installed the latest version of the CLI.
```

Comment:
It is recommended to have the latest version. It's not required though.

Reply:
Fixed.